### ISSUE TITLE
Easy Solution - tracking ACF Options Fields

### ISSUE THREAD

**Issue by jenishshrestha** [gatsbyjs/wp-gatsby#114]
I created an option page called theme option and all the schemas appears correctly in graphql. But whenever I update the options (lets say I changed a image field value or text field value and hit update) they are not being tracked by WpGatsby.

A workaround I am doing right now is edit a random page and update. After this all the changed data gets reflected in graphQL.

Is this a bug or a missing feature?

**Comment by jenishshrestha** [comment]
So, I was looking into code for Action Monitor (even though I am not an expert) because I wanted a solution so badly.

In AcfMonitor.php, I saw two acf action hooks being used to trigger the rebuilding of schemas. Then I tried adding below code in the file.

`add_action(
	'acf/save_post',
	function() {
		$this->trigger_schema_diff(
			[
				'title' => __( 'Update option page', 'WPGatsby' ),
			]
		);
	}
);`

Surprisingly this worked with my test application.

Question: Can this be the solution to the problem without having any other problem?

I can see in the file this feature is in the todo.

`// @todo: Add support for tracking ACF Options Fields.`
